src/toTransform.js

```diff
@@ -36,6 +36,11 @@
   }
 
   function toTransform (properties) {
+    /* W3C spec for what syntax toTransform() outputs:
+       https://drafts.csswg.org/css-transforms/#transform-functions
+       W3C spec for what syntax toTransform() parses:
+       https://drafts.csswg.org/css-transforms-2/#individual-transforms
```

Comment:
We now need to add link(s) for offset properties.
https://drafts.fxtf.org/motion-1/#motion-paths-overview